include/rtc/candidate.hpp

```diff
@@ -25,6 +25,17 @@
 
 namespace rtc {
 
+#if not USE_JUICE
+enum class CandidateType { HOST = 0, SERVER_REFLEXIVE, PEER_REFLEXIVE, RELAYED };
+enum class CandidateTransportType { UDP = 0, TCP_ACTIVE, TCP_PASSIVE, TCP_SO };
```

Comment:
Could you please rename them in camel case for consistency with the other enums? Basically `Host`, `ServerReflexive`, `PeerReflexive`,  `Relayed`, `Udp`, `TcpActive`, TcpPassive`, and `TcpSo`.
After that it's good to go!